contrib/miniselect/.clang-format

```diff
@@ -0,0 +1 @@
+BasedOnStyle: Google
```

Comment:
Submodule should be more consistent with other contribs.

Reply:
Ok, I just looked that pdqsort was moved out of contrib and wondering why